client/scripts/com/monsters/display/ImageCache.as

```diff
@@ -208,7 +209,7 @@ package com.monsters.display
          var l:Loadable = queue;
          l.loadState = LOADING;
          req_str = l.shouldPrepend ? prependImagePath + l.key : l.key;
-         l.loader.load(new URLRequest(req_str));
+         l.loader.load(new URLRequest(req_str), new LoaderContext(true));
```

Comment:
Out of curiosity whats this do?

Reply:
it's for the crossdomain.xml, just telling the load function to the get crossdomain.xml of the cdn url. Like in the [AuthForm](https://github.com/bym-refitted/backyard-monsters-refitted/blob/9c0227998db95b4bbba5e517e4bd0c9947e73d5f/client/scripts/AuthForm.as#L138)

Reply:
Nice one thank you